jena-core/src/main/java/org/apache/jena/vocabulary/ACL.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.vocabulary;
+
+import org.apache.jena.graph.Node ;
+import org.apache.jena.rdf.model.Property ;
+import org.apache.jena.rdf.model.Resource ;
+import org.apache.jena.rdf.model.ResourceFactory ;
+
+public class ACL {
+    
+/**
+*    Basic Access Control ontology.
+*    https://www.w3.org/wiki/WebAccessControl
+*/
+
+    public static final String uri = "http://www.w3.org/ns/auth/acl#";
+
+    /** returns the URI for this schema
+        @return the URI for this schema
+    */
+    public static String getURI()
+        { return uri; }
+
+    protected static final Resource resource( String local )
+        { return ResourceFactory.createResource( uri + local ); }
+
+    protected static final Property property( String local )
+        { return ResourceFactory.createProperty( uri, local ); }
+
+    public static Resource APPEND = Init.Append();
+    public static Resource WRITE = Init.Write();
+    public static Resource CONTROL = Init.Control();
+    public static Resource ACCESS = Init.Access();
+    public static Resource AUTHORIZATION = Init.Authorization();
+    public static Resource AUTHENTICATEDAGENT = Init.AuthenticatedAgent();
+    public static Resource ORIGIN = Init.Origin();
+    public static Resource READ = Init.Read();
+    public static final Property accessControl = Init.accessControl();
+    public static final Property accessTo = Init.accessTo();
+    public static final Property delegates = Init.delegates();
+    public static final Property mode = Init.mode();
+    public static final Property agentClass = Init.agentClass();
+    public static final Property origin = Init.origin();
+    public static final Property _default = Init._default();
+    public static final Property agent = Init.agent();
+    public static final Property agentGroup = Init.agentGroup();
+    public static final Property accessToClass = Init.accessToClass();
+    public static final Property defaultForNew = Init.defaultForNew();
+    public static final Property owner = Init.owner();
+
+    public static class Init {
```

Comment:
This shouldn't need the "Init" 

Use of "Init" is needed in RDF.java uniquely to stop class preliminary initialization problems within jena-core. It arises because Jena internally uses `RDF` constants. Once jena-core initialization (which is the earliest) then everything should be normal.

If you encountered a problem, we need to track it down.

Reply:
No problems.  Was following the code for RDF.  I removed the Init function.  Added the final new line.